src/main/resources/templates/admin.html

```diff
@@ -203,16 +202,15 @@ <h2 class="card-title mt-md-1 mt-0 mb-1">Manage Person</h2>
                                     </div>
                                 </div>
                             </form>
-                            <!--message for no groupings selected-->
-
-                            <div ng-show="!errorDismissed2 && !loading && emptySelect"
-                                 class="alert alert-danger form-alert fade show"
-                                 role="alert">
-                                <p th:text="#{screen.message.modal.search.noChecks}"></p>
-                            </div>
-
                         </div>
-
+                        <div class="flex-break"></div>
```

Comment:
This flex-break. Get rid of it here and in our css file.

in bootstrap you can do this: 
<div class="w-100"></div>